test/index.test.js

```diff
@@ -1148,7 +1168,8 @@ jobs:
 
             return scm.decorateUrl({
                 scmUri,
-                token: 'mytokenfortesting'
+                token: 'mytokenfortesting',
+                scmContext: 'github.com'
```

Comment:
I see that it's passing in additional param `scmContext`, but the function `decorateUrl` itself is not changed?